Author: Luigi Gangitano <luigi@debian.org>
Description: Change default file locations for debian
Index: squid3-3.1.18/src/Makefile.in
===================================================================
--- squid3-3.1.18.orig/src/Makefile.in	2011-12-26 22:10:23.000000000 +0100
+++ squid3-3.1.18/src/Makefile.in	2011-12-26 22:35:27.000000000 +0100
@@ -2045,7 +2045,7 @@
 DEFAULT_PREFIX = $(prefix)
 DEFAULT_CONFIG_DIR = $(sysconfdir)
 DEFAULT_CONFIG_FILE = $(DEFAULT_CONFIG_DIR)/squid.conf
-DEFAULT_MIME_TABLE = $(DEFAULT_CONFIG_DIR)/mime.conf
+DEFAULT_MIME_TABLE = $(datadir)/mime.conf
 DEFAULT_DNSSERVER = $(libexecdir)/`echo dnsserver | sed '$(transform);s/$$/$(EXEEXT)/'`
 DEFAULT_SSL_CRTD = $(libexecdir)/`echo ssl_crtd  | sed '$(transform);s/$$/$(EXEEXT)/'`
 DEFAULT_LOG_PREFIX = $(DEFAULT_LOG_DIR)
@@ -2054,7 +2054,7 @@
 DEFAULT_STORE_LOG = $(DEFAULT_LOG_PREFIX)/store.log
 DEFAULT_PID_FILE = $(DEFAULT_PIDFILE)
 DEFAULT_NETDB_FILE = $(DEFAULT_LOG_PREFIX)/netdb.state
-DEFAULT_SWAP_DIR = $(localstatedir)/cache
+DEFAULT_SWAP_DIR = $(localstatedir)/spool/squid3
 DEFAULT_SSL_DB_DIR = $(localstatedir)/lib/ssl_db
 DEFAULT_PINGER = $(libexecdir)/`echo pinger | sed '$(transform);s/$$/$(EXEEXT)/'`
 DEFAULT_UNLINKD = $(libexecdir)/`echo unlinkd | sed '$(transform);s/$$/$(EXEEXT)/'`
