To fix #568360 and #568537, don't limit local part size.
Index: b/program/include/main.inc
===================================================================
--- a/program/include/main.inc	2009-10-31 14:44:19.000000000 +0100
+++ b/program/include/main.inc	2010-07-17 17:33:25.000000000 +0200
@@ -1427,7 +1427,7 @@
     return false;
 
   // Check that there's one @ symbol, and that the lengths are right
-  if (!preg_match('/^([^@]{1,64})@([^@]{1,255})$/', $email, $email_array))
+  if (!preg_match('/^([^@]+)@([^@]{1,255})$/', $email, $email_array))
     return false;
 
   // Check local part
